test/Microsoft.Azure.Devices.Edge.Test.Common/Platform.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Test.Common
+{
+    using System.Runtime.InteropServices;
+
+    public static class Platform
+    {
+        public static IEdgeDaemon CreateEdgeDaemon(string installerPath) => IsWindows()
```

Comment:
```suggestion
        public static IEdgeDaemon InstantiateEdgeDaemon(string installerPath) => IsWindows()
```

Reply:
I don't see a clear benefit to the word "instantiate" vs "create"